bitcoin/src/blockdata/transaction.rs

```diff
@@ -698,15 +698,20 @@ impl Transaction {
                 .collect(),
             output: self.output.clone(),
         };
-        cloned_tx.txid().into()
+        cloned_tx.compute_txid().into()
     }
 
+    #[deprecated(since = "0.31.0", note = "use compute_txid() instead")]
+    #[doc(alias = "compute_txid")]
```

Comment:
I thought we should add `alias = "txid"` on the `compute_txid`, so when someone types `tx.txid` the LSP will suggest `compute_txid`? But I might be wrong. @Kixunil might know better.

Reply:
ah oops, yeah that's an easy change.  I don't currently use LSP so I don't care either way.

Reply:
@dpc yes, needs to be the other way around.